### ISSUE TITLE
`name:` vs. `short_name:`

### ISSUE THREAD

**Issue by mbland** [18F/testing-grouplet#16]
Just noticed that we're using `name:`, `full_name:`, and `short_name:` in our `.about.yml` schema. When I import this data into this repo using `scripts/projects/update_projects.rb`, this leads to an inconsistency between what's currently done in 18F/data-private, which is to have `name:` effectively be the `short_name:`.

I'm going to update `update_projects.rb` to munge `short_name:` into `name:` for now, but I'd propose updating our `.about.yml` schema to use `name:` instead, lest others have better ideas.

cc: @arowla @monfresh @jmcarp

**Comment by arowla** [comment]
Yeah, I felt similarly that we were getting a proliferation of too many
names... I think it came out of the Testing Grouplet's existing data in
various places... full_name: "18F Testing Grouplet", name: "Testing
Grouplet", and short_name: "testing". Agreed that should probably be cut
down to two with short_name getting promoted to name.

On Thu, Aug 6, 2015 at 11:09 AM, Mike Bland notifications@github.com
wrote:

> Just noticed that we're using name:, full_name:, and short_name: in our
> .about.yml schema. When I import this data into this repo using
> scripts/projects/update_projects.rb, this leads to an inconsistency
> between what's currently done in 18F/data-private, which is to have name:
> effectively be the short_name:.
> 
> I'm going to update update_projects.rb to munge short_name: into name:
> for now, but I'd propose updating our .about.yml schema to use name:
> instead, lest others have better ideas.
> 
> cc: @arowla https://github.com/arowla @monfresh
> https://github.com/monfresh @jmcarp https://github.com/jmcarp
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/18F/testing-grouplet/issues/16.

## 

Alison Rowland
Technical Lead, https://fbopen.gsa.gov | 18F
202-317-0124
